model/histogram/float_histogram.go

```diff
@@ -130,6 +130,60 @@ func (h *FloatHistogram) String() string {
 	return sb.String()
 }
 
+// PromQlExpression returns a PromQL expression that evaluates to this histogram when used in promtool.
+func (h *FloatHistogram) PromQlExpression() string {
+	trim := func(s string) string {
+		for {
+			found := false
+			s, found = strings.CutSuffix(s, "0")
```

Comment:
Looks like this causes the following for previous Go version tests:

```
model/histogram/float_histogram.go:138:23: undefined: strings.CutSuffix
```

Reply:
Luckily, we don't need the trim function, see my comment above.